pilot/pkg/xds/delta.go

```diff
@@ -471,16 +471,22 @@ func (s *DiscoveryServer) pushDeltaXds(con *Connection, push *model.PushContext,
 		return err
 	}
 
-	// Some types handle logs inside Generate, skip them here.
-	if !gen.Metadata().LogsDetails {
-		if log.DebugEnabled() {
-			// Add additional information to logs when debug mode enabled
-			log.Infof("%s: PUSH for node:%s resources:%d size:%s nonce:%v version:%v",
-				v3.GetShortType(w.TypeUrl), con.proxy.ID, len(res), util.ByteCount(ResourceSize(res)), resp.Nonce, resp.SystemVersionInfo)
-		} else {
-			log.Infof("%s: PUSH for node:%s resources:%d size:%s",
-				v3.GetShortType(w.TypeUrl), con.proxy.ID, len(res), util.ByteCount(ResourceSize(res)))
+	ptype := "PUSH"
+	info := ""
+	if logdata != nil {
+		if logdata.Incremental {
+			ptype = "PUSH INC"
```

Comment:
incremental should be log.Debug level, not info, which I think is changed here?

Reply:
Oh. Yes. I did not notice that. Fixed now.